pkg/postrender/exec.go

```diff
@@ -38,12 +39,20 @@ func NewExec(binaryPath string) (PostRenderer, error) {
 	if err != nil {
 		return nil, err
 	}
-	return &execRender{fullPath}, nil
+	return &execRender{fullPath, []string{}}, nil
+}
+
+func NewExecWithArgs(binaryPath string, args []string) (PostRenderer, error) {
```

Comment:
I realized there is a better way to be backwards-compatible.  Instead of adding a new function `NewExecWithArgs()`, you can instead change the existing one, from `func NewExec(binaryPath string) (PostRenderer, error)` to `func NewExec(binaryPath string, args ...string) (PostRenderer, error)`.

This approach allows to call `NewExec(path, args...)` to specify arguments, but also allows calling `NewExec(path)` like before.

Sorry about not having thought about it before.

Reply:
i got it,  this is best way! i did not found yet, very thanks for u suggestion, i will fix those later.